### ISSUE TITLE
Support gap used between the draggables in flexbox

### ISSUE THREAD

**Issue by Darhagonable** [hello-pangea/dnd#432]
### Expected behavior
When using `gap`, with `flex` it should work properly and similar to `margin` property.

### Actual behavior
When dragging the placeholder is too small and when dropping it lands slightly off place and then snaps back to its correct position.

### Demo
https://codesandbox.io/s/vertical-list-with-flex-gap-mt4ugc?file=/src/App.tsx

**Comment by 100terres** [comment]
Hi @Darhagonable :wave:

Thank you for opening this ticket!

Could you properly document the issue in the description of this ticket to avoid back and forth between `@hello-pangea/dnd` and [`react-beautiful-dnd`](https://github.com/atlassian/react-beautiful-dnd/) repos.

**Comment by Darhagonable** [comment]
@100terres I have updated the OP and added a codesandbox of the issue

**Comment by clicktodev** [comment]
This a much needed bug fix. Gap support is very much needed as it's supported by all major browsers.
Can we get some movement on this?

**Comment by mguzm4n** [comment]
Uhm, I think gap is not working yet isn't it? - Added a thumb up.

**Comment by rick-25** [comment]
Yep, it's still not working

**Comment by clicktodev** [comment]
For now, It's a simple fix with margin bottom on each element.

**Comment by jwu-ice** [comment]
It's simple fix with margin bottom and last-child: margin bottom: 0
but same problem with drag and drop last child position

**Comment by korneef** [comment]
Yes, it has a problem if I need to use another margin for last-child component. Or different margin for any component of list. But it's more actual for last/first-child
